imports/startup/client/routes.js

```diff
@@ -255,7 +255,7 @@ FlowRouter.route('/underwriting', {
 FlowRouter.route('/resend', {
   name: 'resend',
   action: async function() {
-    await import('../../ui/components/resend/resend.js').then(function() {
+    await import('../../ui/components/static_pages/resend.js').then(function() {
```

Comment:
Nice. Moving static pages to a shared folder is a good idea. Is resend.js really static, though? I'd think it'd take the email address, but maybe it's just the page saying "we sent you a thing". Can't look now. Anyway, I'll merge as is.